watchtower/wtclient/backup_task.go

```diff
@@ -192,8 +192,22 @@ func (t *backupTask) bindSession(session *wtdb.ClientSessionBody) error {
 		}
 	}
 
-	// All justice transactions have a p2wkh output paying to the victim.
-	weightEstimate.AddP2WKHOutput()
+	// All justice transactions will either use segwit v0 (p2wkh + p2wsh)
+	// or segwit v1 (p2tr).
+	switch len(t.sweepPkScript) {
+	case input.P2WPKHSize:
+		weightEstimate.AddP2WKHOutput()
+
+	// P2WSH and P2TR are the same size: 34 bytes.
+	case input.P2WSHSize:
+		weightEstimate.AddP2WSHOutput()
+
+	default:
+		fmt.Printf("invalid sweep script length %d\n",
```

Comment:
This should probably be a log statement, `fmt.Printf()` is now forbidden by the linter.

Reply:
Stray print statement during debugging, removed!